schemas/variation.json

```diff
@@ -0,0 +1,82 @@
+{
+	"title": "JSON schema for Eightshift blocks used in variation manifest.json",
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"$schema": {
+			"type": "string"
+		},
+		"active": {
+			"type": "boolean",
+			"description": "Is set to false your variation will not be registered and will not show in the block editor. Default: true."
+		},
+		"parentName": {
+			"type": "string",
+			"pattern": "^[^\/][a-z0-9-]*$",
+			"description": "Parent name defines what block is this child of. The name must be exact name as the parent block."
+		},
+		"name": {
+			"type": "string",
+			"pattern": "^[^\/][a-z0-9-]*$",
+			"description": "Name defines the name of variation just like block name."
+		},
+		"title": {
+			"type": "string",
+			"description": "This is the display title for your variation. The block inserter will show this name."
+		},
+		"icon": {
+			"type": "object",
+			"description": "An icon property should be specified to make it easier to identify a block. These can be any of WordPress’ Dashicons (slug serving also as a fallback in non-js contexts). Also you can add our custom icons using es- prefix. All icons can be found here https://infinum.github.io/eightshift-docs/storybook/",
+			"properties": {
+				"background": {
+					"type": "string",
+					"description": "Specifying a background color to appear with the icon e.g.: in the inserter."
+				},
+				"foreground": {
+					"type": "string",
+					"description": "Specifying a color for the icon (optional: if not set, a readable color will be automatically defined)"
+				},
+				"src": {
+					"type": "string",
+					"description": "Specifying an icon for the variation."
+				}
+			},
+			"additionalProperties": false
+		},
+		"description": {
+			"type": "string",
+			"description": "This is a short description for your variation, which can be translated with our translation functions. This will be shown in the block inspector."
```

Comment:
```suggestion
			"description": "A short description for your variation that can be translated with our translation functions. This will be shown in the block inspector."
```